README.md

```diff
@@ -72,6 +75,7 @@ MAIN_URL=www.ohack.org
 AUTH0_AUDIENCE=https://girard-server.herokuapp.com
 AUTH0_DOMAIN=<YOUR AUTH0 TENANT DOMAIN>
 AUTH0_CLIENT_SECRET=<The client ID of your Auth0 backend application>
+AUTH0_ADMIN_ROLE_ID=<The Auth0 role ID for the admin role you created>
```

Comment:
I don't know if they have a consistent format by specificationo, but the ones I see have the form `rol_{16 alphabetical letters}`. It can be found in the Auth0 management dashboard under "User management" > "Roles" > and then clicking on the role whose ID you need.

The [Auth0 management API endpoint](https://auth0.com/docs/api/management/v2#!/Roles/get_role_user) this is needed for has a path `/api/v2/roles/{id}/users` where `id` is the role_id.